core/domain/stats_jobs_one_off.py

```diff
@@ -377,8 +374,6 @@ def prepare_map(cls, item):
                     'state_name': item.state_name,
                     'session_id': item.session_id
                 })
-        else:
```

Comment:
The job will only run over the the models defined in `RecomputeStatisticsOneOffJob.ENTITY_CLASSES_TO_MAP_OVER` and hence this line is unreachable